docker-compose.yml

```
@@ -141,6 +141,7 @@ x-hierarchy:
     - ubuntu-python-sdist-test
     - ubuntu-r
     - ubuntu-r-only-r
+    - ubuntu-swift
```

```suggestion
  - ubuntu-swift
```

(We need to put this between `ubuntu-r-valgrind` and `ubuntu-verify-rc` to keep order.)

---

I see, I will move.